src/map/Map.js

```diff
@@ -1,3 +1,24 @@
+import {
+	bind, extend, setOptions, stamp, indexOf,
+	requestAnimFrame, cancelAnimFrame
+} from '../core/Util';
+
+import {Evented} from '../core/Events';
+import {EPSG3857} from '../geo/crs/CRS.EPSG3857';
+import {Point, toPoint} from '../geometry/Point';
+import {Bounds, toBounds} from '../geometry/Bounds';
+import {LatLng, toLatLng} from '../geo/LatLng';
+import {LatLngBounds, toLatLngBounds} from '../geo/LatLngBounds';
+
+import {
+	any3d as isAny3D,

```

Comment:
at least within iD and Studio, we've tried to avoid `as` statements so that it's easier to track down code across the whole codebase. Minor nitpick, and not necessarily something you want to adopt, but I think it's worked well.